base/logger.go

```diff
@@ -34,3 +38,45 @@ func logCollationWorker(collateBuffer chan string, logger *log.Logger, bufferSiz
 		}
 	}
 }
+
+func runLogDeletion(logDirectory string, logLevel string, maxSize int) {
```

Comment:
We'll want to return errors from this, not `fmt.Println` them inside.

```suggestion
func runLogDeletion(logDirectory string, logLevel string, maxSize int) (err error) {
```

Reply:
Thats my bad...